README.md

```diff
@@ -145,12 +145,36 @@ npx ngrok http 3000
   - Add in Allowed Callback URLs: `http://localhost:3000/api/auth/callback`
   - Add in Allowed logout URLs : `http://localhost:3000`
 
+## A0deploy CLI
+
+Setup the application with ```tenanl.yaml``` file
+**Export data**
```

Comment:
add line break

Reply:
Note: apparently the original author didn't mind airing the markdown with linebreak